### ISSUE TITLE
project have two @angular/**.

### ISSUE THREAD

**Issue by withOutUndo** [inorganik/countUp.js-angular2#15]
```
[x] Bug
[ ] Build issue
[ ] Feature request
```

#### Version info 

```
Angular:5.2.11
Angular CLI:6.0.8
countup.js-angular2: 6.0.1
```

#### Description
error info:
ERROR in node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error TS2451: Cannot redeclare
 block-scoped variable 'ngDevMode'.
node_modules/countup.js-angular2/node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error
TS2451: Cannot redeclare block-scoped variable 'ngDevMode'.

this project have node_module/countup.js-angular2/node_modules/@angular/core, but I project have node_modules/@angular/core. so I remove the node_module/countup.js-angular2/node_modules/** fix this issue.

**Comment by newzhongdan** [comment]
how solve the problem with ionic 3?

**Comment by inorganik** [comment]
Hi, I think this is resolved, I just pushed 6.0.2 which has angular as a peer dependency. Please try it and if you still have trouble we can reopen.
